### PR TITLE
WIP: Fix Chart Settings

### PR DESCRIPTION
**Pull request by chakrabortyr**
WIP: Don't review yet, just want Shippable to run my tests.

## Description
Removes spurious JSON.encode/decode calls for chart settings in the Usage/Metric Explorer Tabs

## Motivation and Context
Already perfectly valid JSON, encoding/decoding is redundant. 
 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: classes/DataWarehouse/Access/Usage.php [hunk 1]
```diff
@@ -93,18 +93,17 @@ private function getSummaryCharts(XDUser $user) {
                 if (!$statsClass->isVisible()) {
                     continue;
                 }
-                $statUsageChartSettings = $usageChartSettings;
```
Comment (plessbd) [inline classes/DataWarehouse/Access/Usage.php]: why did this get removed?  shouldnt this change just be removing the json encode on line 117

Comment (chakrabortyr) [reply classes/DataWarehouse/Access/Usage.php]: looks like that variable was only ever assigned the value of $usageChartSettings and given a few additional variables. So it seemed redundant, ostensibly at least.

File: tests/artifacts/xdmod/schema/get-menus.spec.json [hunk 2]
```diff
@@ -25,10 +25,10 @@
           "type":"string"
         },
         "defaultChartSettings": {
-          "type":"string"
```
Comment (plessbd) [inline tests/artifacts/xdmod/schema/get-menus.spec.json]: i would like to spec to be updated to have the object be validated for its expected contents as well

File: classes/DataWarehouse/Access/Usage.php (hunk 1 above)
Comment (plessbd) [reply classes/DataWarehouse/Access/Usage.php]: i want to say it had to do with the different settings for the stat vs the groupby but i will need to talk with @ryanrath about why we added that one to be sure.

**Comment by chakrabortyr** [comment]
This is now in #1134

**Pull request closed without merging**